src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($item-size * 8) + ($item-gap * 6);
+  margin: 0 auto;
+  padding-top: 50px;
+  gap: $item-gap;
+
+  @each $day, $count in $days {
+    &--start-day--#{$day} &__day:first-child {
+      margin-left: calc((#{$item-gap} + #{$item-size}) * #{$count});
+    }
+  }
+
+  @for $monthDay from 28 through 31 {
+    $monthDayNew: $monthDay + 1;
+
+    &--month-length--#{$monthDay} :nth-child(n + #{$monthDayNew}) {
+      display: none;
+    }
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    @include itemSize($item-size);
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 1s, transform 0.5s;
+
```

Comment:
remember about code style https://codeguide.academy/html-css.html#css-order

Reply:
what's wrong? 
background
border
transition
<img width="444" alt="image" src="https://user-images.githubusercontent.com/71716013/194022304-dc166bb1-e963-443d-a374-317d6ae799e1.png">

Reply:
add space between border and transition